services/web-server/src/graphql/WorkerManager.graphql

```diff
@@ -0,0 +1,25 @@
+type WMWorkerTypeSummary {
+    name: String!
+    description: String
+    owner: String!
+
```

Comment:
Remove empty line.

Reply:
Can you point me to the schema definition, please?

Reply:
I added these empty lines to visually organize the fields

Reply:
...in the same way it was done here https://github.com/taskcluster/taskcluster/blob/master/services/web-server/src/graphql/AwsProvisionerWorkerTypes.graphql#L179 - I liked that

Reply:
Feel free to also remove the empty line from https://github.com/taskcluster/taskcluster/blob/master/services/web-server/src/graphql/AwsProvisionerWorkerTypes.graphql#L179. By the way, we follow a certain convention for graphql files under the `ui` directory. If a request triggers a second request, we add a newline to show that it's triggering an additional request possibly becoming an expensive operation. For example, in https://github.com/taskcluster/taskcluster/blob/master/ui/src/views/Tasks/ViewTask/task.graphql, `metadata`, `status`, `taskActions`, `decisionTask` all trigger an extra request hence the new line.

Reply:
I won't be removing the empty line in `AwsProvisioner` because it's out of scope of this PR